### PR TITLE
Frist version

### PR DESCRIPTION
**Pull request by laullon**

### CODE REVIEW

**Comment by laullon** [comment]
@vikramraman @jbau @rkempf42 @panghy

**Comment by vikramraman** [comment]
@laullon Please you use this: https://github.com/sunnylabs/system/blob/master/.idea/codeStyleSettings.xml to format your Java code.

File: src/main/java/com/wavefront/websphere/DafaultServlet.java [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+public abstract class DafaultServlet extends HttpServlet {
+
+    private static final long serialVersionUID = 1L;
+
+    // <editor-fold defaultstate="collapsed" desc="HttpServlet methods. Click on the + sign on the left to edit the code.">
```
Comment (vikramraman) [inline src/main/java/com/wavefront/websphere/DafaultServlet.java]: Remove this commented line?

File: src/main/java/com/wavefront/websphere/InfoServlet.java [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+    protected void processRequest(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException {
+        String wasVersion = "";
+        JSONObject info = new JSONObject();
+
+        MBeanServer server = AdminServiceFactory.getMBeanFactory().getMBeanServer();
+        try {
+            ObjectName queryName = new ObjectName("WebSphere:type=Perf,*");
+            Set<ObjectName> perfNames = server.queryNames(queryName, null);
+            for (ObjectName perfName : perfNames) {
+                wasVersion = perfName.getKeyProperty("version");
```
Comment (vikramraman) [inline src/main/java/com/wavefront/websphere/InfoServlet.java]: Should we break out of the loop here?

**Pull request merged**